Src/java/qdm/schema/qdm.5.3.xsd

```diff
@@ -272,6 +272,7 @@
 					<xs:element name="method" type="dt:Code" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="result" type="dt:Any" minOccurs="0" maxOccurs="1"/>
 					<xs:element name="component" type="Component" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element name="relatedTo" type="dt:Code" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
A question that we've been having with `relatedTo`. This has a type `dt:Code`, but from all of the email exchanges, it sounds like it should be an id. How would this be represented in this file?